addons/explosives/XEH_PREP.hpp

```diff
@@ -37,3 +37,5 @@ PREP(spawnFlare);
 PREP(startDefuse);
 PREP(startTimer);
 PREP(triggerType);
+PREP(excludeMine);
+PREP(stopExcludingMine);
```

Comment:
I would prefer matching names, maybe:
- `excludeMine`
- `includeMine`
or a single function (I think I prefer this):
- `allowDefuse` which takes a boolean

```suggestion
PREP(stopExcludingMine);

```